.github/workflows/build.yml

```diff
@@ -27,3 +27,21 @@ jobs:
         with:
           name: test-results
           path: build/test-results/test/
+  test-reporter:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download
+        uses: actions/download-artifact@v3
+        with:
+          name: test-results
+          path: test-results/test/
+      - name: Display structure of downloaded files
```

Comment:
No need to include this step, can it be removed please? (lines 40 and 41)

Reply:
Thank you for pointing this out. I've just updated it.